astropy/coordinates/tests/test_arrays.py

```diff
@@ -55,6 +55,18 @@ def test_angle_arrays():
         Angle(["04:02:02", "03:02:01", "06:02:01"])
 
 
+def test_angle_from_pyarrow():
+    # handling arbitrary array classes (that do not inherit from np.ndarray)
+    # as well as arbitrary Iterable[str] is non trivial and requires special care
```

Comment:
I see you have added a sentence, but that sentence is a general statement whereas the test is a regression test for a specific bug.
```suggestion
    # Creating Angle instances from some array classes (e.g. pyarrow.array) failed
    # even though creating Quantity instances succeeded.
```